src/transformers/data/data_collator.py

```diff
@@ -138,6 +138,73 @@ def mask_tokens(self, inputs: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]
         # The rest of the time (10% of the time) we keep the masked input tokens unchanged
         return inputs, labels
 
+@dataclass
+class DataCollatorForAlbertPretrain(DataCollatorForLanguageModeling):
```

Comment:
Would it be possible to rename to `DataCollatorForSentenceOrderingPrediction`? It is more explicit, and less constrained to just one model.

Reply:
Can we go for the shorter `DataCollatorForSop` since we use that acronym everywhere else? If the name changes, also make sure to adapt the docstrings first line below.

Reply:
updated